### ISSUE TITLE
This package doesn't install the 'schema' module

### ISSUE THREAD

**Issue by yurivict** [keleshev/schema#311]
Only meta-data is installed:
```
%%PYTHON_SITELIBDIR%%/schema-0.7.6.dist-info/LICENSE-MIT
%%PYTHON_SITELIBDIR%%/schema-0.7.6.dist-info/METADATA
%%PYTHON_SITELIBDIR%%/schema-0.7.6.dist-info/RECORD
%%PYTHON_SITELIBDIR%%/schema-0.7.6.dist-info/WHEEL
%%PYTHON_SITELIBDIR%%/schema-0.7.6.dist-info/top_level.txt
```

Build and installation are done using build/install PEP517-compliant modules.

Installation is done w/in the FreeBSD ports framework.

Version: 0.7.6
Python-3.9
FreeBSD 14.0

**Comment by opoplawski** [comment]
I'm running into the same issue trying to update the Fedora package to 0.7.6.

**Comment by atsampson** [comment]
It's a silent change in the behaviour of `setuptools.find_packages` - it no longer bothers to look for plain modules.

Replacing `packages=find_packages()` with `py_modules=["schema"]` in setup.py appears to fix it.

**Comment by skorokithakis** [comment]
@atsampson could you please file a PR?

**Comment by skorokithakis** [comment]
Released 0.7.7.

**Comment by atsampson** [comment]
Looks like someone beat me to it - thanks for the fix!
